web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -24,6 +24,86 @@
                 />
             </div>
         </div>
+        <div v-if="isNewAccessGrantFlow" class="access-grants__new-title-area">
```

Comment:
I think these cards should be more responsive. When I make the viewport narrow enough, they should probably be stacked rather than organized horizontally. Since it is under a default-false flag, this does not block my approval. Just a thought for future changes.